cirq-core/cirq/optimizers/merge_interactions_to_sqrt_iswap_test.py

```diff
@@ -0,0 +1,187 @@
+from typing import Callable, List
+
+import cirq
+
+
```

Comment:
Please add tests for `TaggedOperation` as well.

Reply:
Done.